packages/gestalt/src/Badge.test.js

```diff
@@ -1,10 +1,18 @@
 // @flow strict
 import React from 'react';
-import renderer from 'react-test-renderer';
+// import renderer from 'react-test-renderer';
```

Comment:
@denamwangi can you remove this comment?

Reply:
🤦‍♀️ thank you